### PR TITLE
Fixes Orion Trail Take 2

### PR DESCRIPTION
**Pull request by Jalleo**
properly the error was a rogue ! in the top of the topic that was suggested by error when it wasn't needed. This doesn't fix the href exploit.

Also re-added myself to the admin.txt

### CODE REVIEW

File: config/admins.txt [hunk 1]
```diff
@@ -92,3 +92,4 @@ Dunc = Game Master
 MMMiracles = Game Master
 bear1ake = Game Master
 CoreOverload = Game Master
+Jalleo = Game Master
```
Comment (Anonus) [inline config/admins.txt]: I KEEP CONFLICTING WITH THIS PLEASE MERGE IT AGAIN SO I STOP CONFLICTING

Comment (Jalleo) [inline config/admins.txt]: Sorry

Comment (Miauw62) [inline config/admins.txt]: wait should i merge this or not???

Comment (Anonus) [inline config/admins.txt]: YES PLEASE DO MERGE IT

**Comment by Miauw62** [comment]
pretty hig hpriority tbh

**Pull request merged**